WORKSPACE

```diff
@@ -89,6 +89,9 @@ bazel_skylib_workspace()
 
 http_archive(
     name = "bazel_gazelle",
+    patches = [
+        "//third_party:com_google_github_bazel_gazelle-internal-in-their-new-location.patch",
```

Comment:
I'd prefer to fix upstream issues first instead of maintaining a patch here. I'll add a note to bazelbuild/bazel-gazelle#803, which is the only reason we have that internal dependency.

Reply:
I think this is a chicken and egg situation. One of these repos will have to go first. I tried creating an `alias` in the `BUILD` file associated with it, but it turns out `load` statements are interpreted outside of the context of `BUILD` files... so I think we're stuck unfortunately. I'm happy to merge these in rapid fire though

1) commit rules_go with the patch
2) update Gazelle to refer to the new version of rules_go
3) update rules_go to refer to the new version of Gazelle